packages/button/src/LionButton.js

```diff
@@ -69,50 +76,33 @@ export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
           padding: 0;
         }
 
-        :host(:focus) {
-          outline: none;
-        }
-
         :host(:focus) .btn {
-          border-color: lightblue;
-          box-shadow: 0 0 8px lightblue, 0 0 0 1px lightblue;
+          outline: 2px solid #bde4ff;
         }
 
         :host(:hover) .btn {
-          background: black;
-          color: whitesmoke;
-        }
-
-        :host(:hover) .btn ::slotted(lion-icon) {
-          fill: whitesmoke;
+          /* up to Application Developers and SubClassers */
         }
 
         :host(:active) .btn,
         .btn[active] {
-          background: grey;
+          /* up to Application Developers and SubClassers */
         }
 
         :host([disabled]) {
           pointer-events: none;
         }
 
         :host([disabled]) .btn {
+          /* up to Application Developers and SubClassers */
           background: lightgray;
-          color: gray;
-          fill: gray;
-          border-color: gray;
+          color: #adadad;
+          fill: #adadad;
```

Comment:
if we start removing unnecessary styles, maybe we can remove even more

this is the difference in Chrome so far between lion and native one:

![image](https://user-images.githubusercontent.com/137844/61800093-aa45b580-ae2c-11e9-94be-58f8bbdd802d.png)

of course the size difference is for A11Y, but I don't see why we need other differences? for cross browser consistency?

Reply:
we discussed this and carefully analyzed all the CSS of the lion-button, and found only 1 thing which is not needed at all for the generic layer - grey `background-color`, but if we remove it, then the demos of `lion-button` will be difficult to understand, so we decided to keep some minimal styling even if not strictly needed for Lion in such cases where removing it is more troubles than keeping it and overriding on the extension layer